Shared/Samples/Augment reality to collect data/README.md

```diff
@@ -33,7 +33,7 @@ When you tap, an orange diamond will appear at the tapped location. You can move
 
 ## About the data
 
-The sample uses a publicly-editable sample tree survey feature service hosted on ArcGIS Online called [AR Tree Survey](https://arcgisruntime.maps.arcgis.com/home/item.html?id=8feb9ea6a27f48b58b3faf04e0e303ed). You can use AR to quickly record the location and health of a tree.
+The sample uses a publicly-editable sample tree survey feature service hosted on ArcGIS Online called [AR Tree Survey](https://www.arcgis.com/home/item.html?id=8feb9ea6a27f48b58b3faf04e0e303ed). You can use AR to quickly record the location and health of a tree.
```

Comment:
I clicked on the link in the readme but I got 403 Forbidden. However, copying and pasting the URL was successful. Actually, this occurs on all the links I tested hosted on AGOL. Could just be me but any idea why?

Reply:
This happens only happens for me if I command click the link (on Firefox). It also happens with the old links, though, so it seems like a separate issue. Have you seen this before @yo1995?

Reply:
I don't see the issue. When verifying links, you can use "open in Incognito".

Reply:
Gotcha, I'm on Chrome but opening in incognito helped.